demo/index.html

```diff
@@ -15,6 +15,11 @@
     <title></title>
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script>
```

Comment:
I would rather not add this to the demo. In polyserve you can just append ?dom=shadow to the url for this, which is how we usually test the other demos